kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -351,6 +371,26 @@ define([
                     });
                     break;
 
+                case 'jobs_retried':
```

Comment:
prelim retry handling

I have tests for this in a branch somewhere but cannot find them at the moment... 😖

Reply:
🤷‍♂️